src/app/containers/CpsTable/CpsTableCell/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { arrayOf, shape } from 'prop-types';
+
+import Blocks from '../../Blocks';
+import CpsText from '../../CpsText';
+
+const StyledTd = styled.td`
+  border-bottom: 1px solid #dbdbdb;
+  padding-top: 8px;
+  padding-bottom: 8px;
+  padding-left: 8px;
```

Comment:
We usually use REMs for units - we have a variable for this :)
```suggestion
  padding-top: ${GEL_SPACING};
  padding-bottom: ${GEL_SPACING};
  padding-left: ${GEL_SPACING};
```